xbmc/cores/AudioEngine/AEResampleFactory.h

```diff
@@ -26,10 +26,18 @@ class IAEResample;
 namespace ActiveAE
 {
 
+/**
+ * Bit options to pass to CAEResampleFactory::Create
+ */
+enum AEStreamOptions {
+  /* This is a quick resample job (e.g. resample a single noise packet) and may not be worth using GPU acceleration */
```

Comment:
It matches:
https://github.com/xbmc/xbmc/blob/master/xbmc/cores/AudioEngine/Interfaces/AEStream.h#L29
I can change if desired.